01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+numbers = (1..20)
+numbers.each do |number| 
```

Comment:
```suggestion
(1..20).each do |number| 
```

でもいいと思います。もしくは

```suggestion
1.upto(20) do |number| 
```

と書くこともできます。